core/src/main/java/discord4j/core/object/automod/AutoModRule.java

```diff
@@ -305,10 +315,12 @@ public int getValue() {
          * @return The type of message.
          */
         public static AutoModRule.EventType of(final int value) {
-            switch (value) {
-                case 1: return MESSAGE_SEND;
-                default: return UNKNOWN;
+            for (AutoModRule.EventType eventType : values()) {
```

Comment:
I think that to follow the library standards, you should keep a switch here

Reply:
Yeah but this logic are in another places too, like others enums in the AutoMod thing

Reply:
We should choose one logic and apply it everywhere I think, to keep everything the same.
 And switch statements are more efficient so I would privilege them.

Reply:
by the moment i keep this here and in the future mark a standar for this things.

Reply:
LGTM, personally I use the switch variant on all of my PRs